65.md

```diff
@@ -6,71 +6,53 @@ Relay List Metadata
 
 `draft` `optional` `author:mikedilger`
 
-A special replaceable event meaning "Relay List Metadata" is defined as an event with kind `10002` having a list of `r` tags, one for each relay the author uses to either read or write to.
+Defines a replaceable event using `kind:10002` to advertise a user's preferred relays to others. 
 
-The primary purpose of this relay list is to advertise to others, not for configuring one's client.
+The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. If the marker is omitted, it means the author uses the relay for both purposes.
 
-The `content` is not used and SHOULD be an empty string.
+The `content` is not used.
 
-The `r` tags can have a second parameter as either `read` or `write`. If it is omitted, it means the author uses the relay for both purposes.
-
-Clients SHOULD, as with all replaceable events, use only the most recent kind-10002 event they can find.
-
-### The meaning of read and write
+```json
+{
+  "kind": 10002,
+  "tags": [
+    ["r", "wss://alicerelay.example.com"],
+    ["r", "wss://brando-relay.com"],
+    ["r", "wss://expensive-relay.example2.com", "write"],
+    ["r", "wss://nostr-relay.example.com", "read"],
+  ],
+  "content": "",
+  ...other fields
+```
 
-Write relays are for events that are intended for anybody (e.g. your followers). Read relays are for events that address a particular person.
+Clients must download all `kind:10002`s from their user's contact list and assemble a relay pool from there.
 
-Clients SHOULD write feed-related events created by their user to their user's write relays.
+## How to Use Read and Write
 
-Clients SHOULD read feed-related events created by another from at least some of that other person's write relays. Explicitly, they SHOULD NOT expect them to be available at their user's read relays. It SHOULD NOT be presumed that the user's read relays coincide with the write relays of the people the user follows.
+When looking for a given event: 
 
-Clients SHOULD read events that tag their user from their user's read relays.
+- Download the author's `kind:10002` and use the WRITE relays
+- Download a tagged user's `kind:10002` and use the READ relays 
```

Comment:
Scratch this.  You never read from somebody's read relays.  You only write there.

Reply:
Don't you read people's read relays when you are visiting their profile? Like to count the zaps they received?

Reply:
nice, that does make sense for the use in profile pages.